Kernel/Config/Files/XML/Ticket.xml

```diff
@@ -10541,6 +10541,39 @@
             <Item ValueType="String" ValueRegex="^\d+%?$">320</Item>
         </Value>
     </Setting>
+    <Setting Name="Frontend::Admin::AdminNotificationEvent::IgnoredAttributes###002-Ticket" Required="0" Valid="1">
```

Comment:
This SysConfig is only used by the EventModule `Kernel::System::Ticket::Event::NotificationEvent`.
Therefore, the configuration should also be saved directly at the registry.

```xml
    <Setting Name="Ticket::EventModulePost###7000-NotificationEvent" Required="0" Valid="1">
        <Description Translatable="1">Sends the notifications which are configured in the admin interface under "Ticket Notifications".</Description>
        <Navigation>Core::Event::Ticket</Navigation>
        <Value>
            <Hash>
                <Item Key="Module">Kernel::System::Ticket::Event::NotificationEvent</Item>
                <Item Key="Transaction">1</Item>
                <Item Key="Event"></Item>
                <Item Key="IgnoredAttributes">
                    <Hash>
                        <Item Key="Recipients">1</Item>
                        <Item Key="SkipRecipients">1</Item>
                        <Item Key="RecipientAgents">1</Item>
                        <Item Key="RecipientGroups">1</Item>
                        <Item Key="RecipientRoles">1</Item>
                        <Item Key="TransportEmailTemplate">1</Item>
                        <Item Key="Events">1</Item>
                        <Item Key="ArticleSenderTypeID">1</Item>
                        <Item Key="ArticleIsVisibleForCustomer">1</Item>
                        <Item Key="ArticleCommunicationChannelID">1</Item>
                        <Item Key="ArticleAttachmentInclude">1</Item>
                        <Item Key="IsVisibleForCustomer">1</Item>
                        <Item Key="Transports">1</Item>
                        <Item Key="OncePerDay">1</Item>
                        <Item Key="VisibleForAgent">1</Item>
                        <Item Key="VisibleForAgentTooltip">1</Item>
                        <Item Key="LanguageID">1</Item>
                        <Item Key="SendOnOutOfOffice">1</Item>
                        <Item Key="AgentEnabledByDefault">1</Item>
                        <Item Key="EmailSecuritySettings">1</Item>
                        <Item Key="EmailSigningCrypting">1</Item>
                        <Item Key="EmailMissingCryptingKeys">1</Item>
                        <Item Key="EmailMissingSigningKeys">1</Item>
                        <Item Key="EmailDefaultSigningKeys">1</Item>
                        <Item Key="NotificationType">1</Item>
                    </Hash>
                </Item>
            </Hash>
        </Value>
    </Setting>
```

Reply:
That wouldn't be as useful as a seperate entry. We can add new Ignored Attributes with a new package for example: 
` <Setting Name="Frontend::Admin::AdminNotificationEvent::IgnoredAttributes###020-Example" Required="0" Valid="1">
<Description Translatable="1">Ignores not ticket related attributes.</Description>
        <Navigation>Frontend::Admin::View::NotificationEvent</Navigation>
        <Value>
            <Hash>
                <Item Key="ExampleAttribute">1</Item>
            </Hash>
        </Value>
    </Setting>
`

Reply:
Hi @LuBroering 

Yes, I thought so.
How often have you had the case that you want to expand this list?
Doesn't it then make sense to update it via code (PackageSetup) or, if useful, to adopt it in the framework?

I am open to a counter proposal.
But still the SysConfig does not match `Frontend::Admin::AdminNotificationEvent`.

Reply:
@dennykorsukewitz 
We expand this list rather often, therefore it would be a cleaner solution to add a new Sysconfig instead of an update via PackageSetup. This way we will just collect all necessary Sysconfigs and IgnoredAttributes.

"But still the SysConfig does not match Frontend::Admin::AdminNotificationEvent."
I'm open for name/naviagtion changes.

Reply:
Deal 👍🏼 

We have agreed on the following name:
`Ticket::Event::NotificationEvent::IgnoredAttributes###001-Framework`

Reply:
I commited the files with the changed name: "Ticket::Event::NotificationEvent::IgnoredAttributes###001-Framework"
:+1: